packages/cli/src/cmds/validator/keys.ts

```diff
@@ -59,9 +59,15 @@ export async function getLocalSecretKeys(
     }
 
     const secretKeys = await Promise.all(
-      keystorePaths.map(async (keystorePath) =>
-        SecretKey.fromBytes(await Keystore.parse(fs.readFileSync(keystorePath, "utf8")).decrypt(passphrase))
-      )
+      keystorePaths.map(async (keystorePath) => {
+        let keystore;
+        try {
+          keystore = Keystore.parse(fs.readFileSync(keystorePath, "utf8"));
+        } catch (e) {
+          throw new Error("Error parsing keystore at " + keystorePath + ": " + (e as Error).message);
```

Comment:
I guess since the key store file does not contain any decrypted secrets, it should be fine to log the location (we recently had a PR that removed similar logging [here](https://github.com/ChainSafe/lodestar/pull/3975)

Suggestion: consider using `template literals` rather than concatenation.

Reply:
We removed the path of the passphrase from the logs since its more sensitive?

Reply:
> We removed the path of the passphrase from the logs since its more sensitive?

Yeah. The advisory that it triggered can be seen [here](https://codeql.github.com/codeql-query-help/python/py-clear-text-logging-sensitive-data/)